README.md

```diff
@@ -27,4 +27,10 @@ This can be done using mkdocs:
 $ mkdocs serve
 ```
 
+To bind to any address or an IP address use:
+
+```
+$ mkdocs serve --dev-addr 0.0.0.00:8000
```

Comment:
The given command actually just binds to the any address, I would hope that people would realize you could put an explicit IP address in there but better to be specific. Perhaps another section.

Reply:
Maybe we need a separate CONTRIBUTING.md with more detailed instructions.  If so, let's wait until I've got the dynamic docs figured out.

Reply:
@jcolp ack